airflow/www/utils.py

```diff
@@ -448,7 +448,42 @@ def __call__(self, field, **kwargs):
         return wtforms.widgets.core.HTMLString(html)
 
 
-class UtcFilterConverter(FilterConverter):
+class UtcDateTimeFilterMixin(object):
+    def clean(self, value):
+        return timezone.make_aware(super(UtcDateTimeFilterMixin, self).clean(value))
```

Comment:
Make aware returns 'default_timezone' is that what you require? I think UTC is better.

Reply:
Yes, this is not working when default_timezone is not in UTC. Will fix.

Reply:
Fixed.